index.html

```diff
@@ -1,16 +1,126 @@
-<!doctype html>
+<!DOCTYPE html>
 
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-  <title>Great Idea - Responsive I</title>
+	<head>
+		<meta charset="utf-8" />
+		<link
+			href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web"
+			rel="stylesheet"
+		/>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Great Idea - Responsive I</title>
 
-  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+		<link
+			href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web"
+			rel="stylesheet"
+		/>
+		<link rel="stylesheet" href="css/index.css" />
+	</head>
 
-</head>
+	<body>
+		<section>
+			<nav>
+				<div>
+					<a href="#">Services</a>
+					<a href="#">Product</a>
+					<a href="#">Vision</a>
+					<a href="#">Features</a>
+					<a href="#">About</a>
+					<a href="#">Contact</a>
+				</div>
+				<div>
+					<img
+						class="logo"
+						src="img/logo.png"
+						alt="Great Idea! Company logo."
+					/>
+				</div>
+			</nav>
+		</section>
+		<header>
+			<div>
+				<h1>Innovation</h1>
```

Comment:
Should only use one ```<h1>``` tag here and then use ```<br>```  line breaks at the end of each work to stack the heading.